model/index.js

```diff
@@ -51,23 +51,20 @@ module.exports = generators.Base.extend({
     initModelName: function () {
       this.skipPromptingAppName = false
 
-      if (this.name) {
+      if (this.options.name) {
         // User passed a desired model name as an argument
-        var validation = validateRequiredName(this.name)
+        var validation = validateRequiredName(this.options.name)
         if (validation === true) {
           // Desired model name is valid, skip prompting for it later
+          debug('Valid model name provided via command-line argument:', this.options.name)
+          this.name = this.options.name
           this.skipPromptingModelName = true
         } else {
           // Log reason for validation failure, if provided
           validation = validation || 'Model name not valid'
           this.log(validation)
         }
       }
-    },
-
-    readConfig: function () {
-      debug('reading config json from: ', this.destinationPath('config.json'))
-      this.config = this.fs.readJSON(this.destinationPath('config.json'))
```

Comment:
This is no longer used and conflicts with Yeoman's `config` property in the base generator. Since we weren't using that feature in this generator it wasn't causing an issue before, but in Yeoman 1.x it now does cause an error.